### ISSUE TITLE
OverlayPanel: add autoHide attribute

### ISSUE THREAD

**Issue by miedrl** [primefaces/primefaces#8546]
**Description**
To guarantee accessibility it has to be possible to hover over an opened overlay panel, mark its text with the mouse and copy said text. At the moment, this does not work as the overlay panel is hidden when the hover is not over the overlay panel's anchor anymore.

**Describe the solution you would like**
The overlay panel has to stay open when hovering over it, I suggest adding the "autoHide" attribute as is already implemented in PrimeReact (see https://primefaces.org/primereact/tooltip/).

- [x] Add `autoHide` property default to `true
- [x] Refactor DOM cleanup
- [x] Add `destroy` method to Selenium widget
- [x] Test `destroy` in Integration Test
- [x] Add showcase example

**Comment by melloware** [comment]
Test case:
[pf-7580.zip](https://github.com/primefaces/primefaces/files/8223502/pf-7580.zip)

**Comment by miedrl** [comment]
> Test case: [pf-7580.zip](https://github.com/primefaces/primefaces/files/8223502/pf-7580.zip)

Can you tell me how to run this in command line?

**Comment by melloware** [comment]
`mvn  clean jetty:run` and navigate to http://localhost:8080/primefaces-test/

you may have to change pom.xml from 12.0.0-SNAPSHOT back to 11.0.0 release.
